make/mt-config/mt-presets.mk

```diff
@@ -6,27 +6,24 @@
 include make/mt-config/mt-target-platform.mk
 
 # CONFIG presets
-CONFIG_POSIX_MAX			=bsp_posix-posix_fs-posix_platform-tls-senml-control_topic-memory_limiter
-CONFIG_POSIX_MAX_THREADING  =bsp_posix-posix_fs-posix_platform-tls-senml-control_topic-threading-memory_limiter
-CONFIG_POSIX_MID			=bsp_posix-posix_fs-posix_platform-tls-senml-control_topic
-CONFIG_POSIX_MID_UNSECURE   =bsp_posix-posix_fs-posix_platform-senml-control_topic
-CONFIG_POSIX_MIN			=bsp_posix-posix_fs-posix_platform-tls
+CONFIG_POSIX_MAX				=bsp_posix-posix_fs-posix_platform-tls_bsp-senml-control_topic-memory_limiter
+CONFIG_POSIX_MAX_THREADING		=bsp_posix-posix_fs-posix_platform-tls_bsp-senml-control_topic-threading-memory_limiter
```

Comment:
in sublime text everything is fine. I think "only" this github reader shows this "wrong"? Please check in your editor whether it's ok.